src/backtrace/mod.rs

```diff
@@ -129,15 +129,15 @@ with_backtrace! {
     impl Debug for Backtrace {
         fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
             if let Some(bt) = self.internal.as_backtrace() {
-                bt.fmt(f)
+                Debug::fmt(bt, f)
             } else { Ok(()) }
         }
     }
 
     impl Display for Backtrace {
         fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
             if let Some(bt) = self.internal.as_backtrace() {
-                bt.fmt(f)
+                Display::fmt(bt, f)
```

Comment:
This will not work with old versions of backtrace.  This should be `Debug::`, or the minimal version in `Cargo.toml` should be raised.